platform/darwin/src/http_file_source.mm

```diff
@@ -196,12 +195,11 @@ BOOL isValidMapboxEndpoint(NSURL *url) {
 }
 
 MGL_APPLE_EXPORT
-NSURL *resourceURLWithAccountType(const Resource& resource, NSInteger accountType) {
+NSURL *resourceURL(const Resource& resource) {
     
     NSURL *url = [NSURL URLWithString:@(resource.url.c_str())];
-    
-#if TARGET_OS_IPHONE || TARGET_OS_SIMULATOR
```

Comment:
This conditional compilation can be removed, but the addition of the `sku` parameter still needs to be conditionalized to only occur on iOS: #16455.